src/App.js

```diff
@@ -5,15 +5,25 @@ import Header from './components/Header/Header';
 import Footer from './components/Footer/Footer';
 import CentrePanel from './components/Centre/CentrePanel';
 
+import { BrowserRouter } from "react-router-dom";
+import { Switch, Route } from 'react-router-dom';
+import Success from './components/Result/Success';
+import Failure from './components/Result/Failure';
+
 function App() {
   return (
     <div className="App">
-      <Header></Header>
-      <CentrePanel>
-      </CentrePanel>
-      <Footer></Footer>
+      <Header/>
```

Comment:
This could be indented a bit to ease to read

Reply:
@eamonnprwalsh that was the old change.